go.mod

```diff
@@ -22,4 +22,4 @@ require (
 	gotest.tools/v3 v3.0.2
 )
 
-replace github.com/johannesboyne/gofakes3 => github.com/igungor/gofakes3 v0.0.5
\ No newline at end of file
+replace github.com/johannesboyne/gofakes3 => github.com/igungor/gofakes3 v0.0.5
```

Comment:
We can now switch to https://github.com/johannesboyne/gofakes3 (upstream). With the latest gofakes3 tests will pass. 

Could you please remove the `replace` directive  and set gofakes3 version to 4b6b67a85d38ec538153d0f21b36378a38edb143(latest commit)?

Reply:
I switched to the latest commit, then run `go mod vendor`. Running on my computer (macOS 11.2.3), the test is passed but failed on ci/test

<img width="517" alt="Screen Shot 2021-04-15 at 23 39 26" src="https://user-images.githubusercontent.com/13713865/114907239-13612e00-9e45-11eb-8573-c2e6e1439195.png">

Reply:
This is also unexpected for us. We are trying to figure out the root cause of these flaky tests. I will update the issue as soon as we find something.

Reply:
Hi @khacminh 

Could you please update gofakes3 dependency to https://github.com/igungor/gofakes3/releases/tag/v0.0.6  by re-adding the replace directive?

```
replace github.com/johannesboyne/gofakes3 => github.com/igungor/gofakes3 v0.0.6
```

We decided to continue to use our fork, because with gofakes3 upstream, tests are failing randomly while running in parallel.

Reply:
Yay! The tests are passed now

Reply:
hi @ilkinulas 
I also revert the go.mod file to minimize the file changes. Re-run the tests and they are passed.